tests/misc-utils.h

```diff
@@ -41,6 +41,12 @@ object_array_new (void)
   return g_ptr_array_new_with_free_func (g_object_unref);
 }
 
+static inline gboolean
+flag_check (guint flags, guint flag)
+{
+  return (flags & flag) == flag;
```

Comment:
This is equivalent to `(flags & flag)`, so I don’t see the point in having the helper function for this.